components/newBillHomePageButtons.jsx

```diff
@@ -0,0 +1,10 @@
+export default function newBillHomePageButtons({ text, iconName }){
```

Comment:
Please add a space between `({ text, iconName })` and the curly brace like `({ text, iconName }) {`